code/modules/antagonists/clockcult/clock_structures/stargazer.dm

```diff
@@ -0,0 +1,70 @@
+#define STARGAZER_RANGE 3 //How many tiles the stargazer can see out to
+#define STARGAZER_POWER 15 //How many watts will be produced per second when the stargazer sees starlight
```

Comment:
change this to 3, since these are not only spammable, but we really ought to encourage clock cult to get power by other means

Reply:
I mean their other means is well...apcs, this was to be another means.

Reply:
I even jacked up the power requisites by 10,000 for each of them

Reply:
However a better solution to this instead of nerf it to the ground is buffing the power cost to make them and dropping it to 7

Reply:
making a far better investment but not as spammable as prior.